src/CodeGen_Internal.cpp

```diff
@@ -690,6 +692,7 @@ void get_target_options(const llvm::Module &module, llvm::TargetOptions &options
     options.FloatABIType =
         use_soft_float_abi ? llvm::FloatABI::Soft : llvm::FloatABI::Hard;
     options.RelaxELFRelocations = false;
+    options.MCOptions.ABIName = mabi;
```

Comment:
UPDATE: just realized that our buildbots are currently only testing PRs against LLVM 12 and 13, so this may not be easy to test. I'll work on fixing that.

Reply:
UPDATE #2: I tested locally against LLVM 10 and LLVM 11 and this seems to be fine as-is for those, so ignore this thread of comments.

Reply:
Thanks!